tdrs-frontend/package.json

```diff
@@ -39,6 +39,7 @@
     "test:accessibility": "concurrently -k -s first 'REACT_APP_PA11Y_TEST=true npm run start:ci' 'wait-on http://localhost:3000/ && npm run pa11y-ci'",
     "test:e2e": "cypress open",
     "test:e2e-ci": "cypress run --headless -b chrome --env cypressToken=${CYPRESS_TOKEN}",
+    "test:e2e-develop": "cypress run --headless -b chrome --config baseUrl=https://tdp-frontend-develop.app.cloud.gov --env cypressToken=${CYPRESS_TOKEN},apiUrl=https://tdp-backend-develop.app.cloud.gov/v1",
```

Comment:
Hmm, this is going to need updating almost immediately. I'm sure it is functional now but we have two changes incoming:

1. Nginx Config #1544, all traffic will be going through frontend
2. New staging/develop FQDNs #2385

Reply:
Converted to ENV VARS and I have updated the env vars.